cli/src/main/java/com/liferay/blade/cli/command/DeployRemoteArgs.java

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
@christopherbryanboyd  can you add this to a new project at `:extensions:deploy-remote-command` and then we can give people instructions of how to install it
`blade extensions install <url>` 
we might need to go ahead and publish registry.